config.cmake

```diff
@@ -454,7 +454,7 @@ config_string(
 )
 
 config_string(
-    KernelStaticMaxBudgetUs KERNEL_STATIC_MAX_BUDGET_US
+    KernelStaticMaxPeriodUs KERNEL_STATIC_MAX_PERIOD_US
```

Comment:
the effort to impact ration of this change seems very poor. But if it's already done...

Reply:
Micheal suggested it from the side of verification, I normally wouldn't suggest a change of this kind at this point.